.clang-format

```diff
@@ -86,5 +86,18 @@ SpacesInCStyleCastParentheses: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
 Standard: c++11
+StatementMacros:
+  - __thrust_exec_check_disable__
+  - __host__
+  - __device__
+  - __global__
+  - THRUST_DEVICE_FUNCTION
+  - THRUST_HOST_FUNCTION
+  - THRUST_FUNCTION
+  - THRUST_HOST_DEVICE
+  - THRUST_NAMESPACE_BEGIN
```

Comment:
`NamespaceMacros` or `MacroBlock[Begin|End]` seem more relevant for these ([link](https://clang.llvm.org/docs/ClangFormatStyleOptions.html)).